source/_docs/pantheon-yml.md

```diff
@@ -111,6 +72,25 @@ Add `drush_version` to the top level of the `pantheon.yml` file to configure the
 drush_version: 8
 ```
 For more information, see [Managing Drush Versions on Pantheon](/docs/drush-versions).
+
+
+### Custom Upstreams and pantheon.upstream.yml
+Add a `pantheon.upstream.yml` file to your organization's custom upstream to set default settings for all downstream sites. Downstream sites can override the defaults at the site-level with pantheon.yml. Quicksilver workflows are not supported in pantheon.upstream.yml.
+
+Create the `pantheon.upstream.yml` in the root of the custom upstream's codebase. Help avoid merge conflicts by adding pantheon.upstream.yml to the upstream's .gitignore. Example: [Adding pantheon.upstream.yml to Drops-7](https://github.com/pantheon-systems/WordPress/pull/123/files)
+
+Because pantheon.upstream.yml is in the .gitignore use git force-add to commit:
```

Comment:
Suggest adding a new h3 header here so this can be a section on how to edit and distribute changes to `pantheon.upstream.yml` -- I also think we need to keep something similar to the original text because it specified that changes to this file are distributed downstream